### PR TITLE
Add a blog for using kubekey to add master nodes for HA

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

### CODE REVIEW

File: content/en/blogs/add-master-for-ha-using-kubekey.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+| 172.16.0.6 | master3   | master, etcd | CentOS 7.5, 8 Core, 8 G Memory, 50 G Disk |
+| 172.16.0.7 | worker3   | worker       | CentOS 7.5, 8 Core, 8 G Memory, 50 G Disk |
+
+![node-info](https://ap3.qingstor.com/kubesphere-website/docs/ha-architecture-node-info.jpg)
+
+For more information about requirements for nodes, network, and dependencies, [see this article](https://kubesphere.io/blogs/install-kubernetes-using-kubekey/#node-requirements).
+
+## Prepare Load Balancers
+
+You can use any cloud load balancers or hardware load balancers (e.g. F5). In this example, I have an internal load balancer with a listener that listens on port `6443` (`api-server`) and an external load balancer with a listener that listens on the port of the Kubernetes dashboard. If your cluster is deployed on bare metal, you can use [Porter](https://github.com/kubesphere/porter) to provide load balancing solutions. [Porter](https://landscape.cncf.io/?selected=porter-lb) is a certified cloud-native tool included in the CNCF Landscape.
```
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply content/en/blogs/add-master-for-ha-using-kubekey.md:51]: Ref: https://kubesphere.com.cn/forum/d/1566-kubernetes-keepalived-haproxy

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]

File: content/en/blogs/add-master-for-ha-using-kubekey.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sherlock113) [reply content/en/blogs/add-master-for-ha-using-kubekey.md:51]: I just have one question: 
40401611219409_ pic
So the Porter doc is incorrect here?

[the remark this answers is not in the document]
Comment (Sherlock113) [reply content/en/blogs/add-master-for-ha-using-kubekey.md]: I will add the example of using keepalived and haproxy in another blog.

[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply content/en/blogs/add-master-for-ha-using-kubekey.md]: I think we have clarified its concepts.

[the remark this answers is not in the document]
Comment (Sherlock113) [reply content/en/blogs/add-master-for-ha-using-kubekey.md]: Already changed.

[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply content/en/blogs/add-master-for-ha-using-kubekey.md]: > I will add the example of using keepalived and haproxy in another blog.

ok.

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**